com.github.simonwhitaker.Gibo.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.github.simonwhitaker.Gibo
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: gibo
+finish-args:
+  - --filesystem=~/.gitignore-boilerplates
```

Comment:
Any reasons for not using `$XDG_DATA_HOME` or `$XDG_CONFIG_HOME`?

Reply:
That's where the tool puts its stuff. There is no way to influence that.